lms/djangoapps/verify_student/views.py

```diff
@@ -1004,10 +1026,15 @@ def results_callback(request):
         log.debug("Approving verification for %s", receipt_id)
         attempt.approve()
         status = "approved"
+        _set_requirement_status(attempt, 'reverification', 'satisfied')
```

Comment:
@rlucioni yes this is intended as in ```CreditRequirementStatus``` ```satisfied``` and ```failed``` statuses are used whereas here ```approved``` or ```denied``` are used.